pkg/daemon/file_writers.go

```diff
@@ -155,6 +155,11 @@ func writeDropins(u ign3types.Unit, systemdRoot string, isCoreOSVariant bool) er
 // it doesn't fetch remote files and expects a flattened config file.
 func writeFiles(files []ign3types.File) error {
 	for _, file := range files {
+		if file.Path == caBundleFilePath {
+			// TODO remove this special case once we have a better way to do this
+			glog.V(4).Infof("Skipping file %s during writeFiles", caBundleFilePath)
```

Comment:
I'm assuming...Hypershift just doesn't need to sync these certificates? 

Since we don't write this file via writeFiles anymore, and HypershiftConnect doesn't wire up the certificate writer/cclister, Hypershift won't ever be able update this bundle, will it? Does that matter? 

(It's possible I'm missing something :smile: )

Reply:
Fair question! So this would have broken HyperShift inplace upgrades if certificates rotate, but the HyperShift bundle is only 2 certs both valid for 10 years and have no rotation mechanism, such that in practice it shouldn't happen. And for replace/inplace's bootstrap, the bootstrap MCS will take care of serving it the right cert.

We definitely should consider that use case for a more comprehensive solution, but for now, this shouldn't break them. I can add that as a comment or I can make the jira issue more clear regarding that

Reply:
Oh hey, I remember us talking about that, I think I just didn't connect all the dots. Sorry. I probably should have just looked at the certs.  Thank you.  :smile: 

```
        Issuer: OU=openshift, CN=kube-csr-signer
        Validity
            Not Before: Mar  8 00:21:04 2023 GMT
            Not After : Mar  5 00:21:04 2033 GMT
        Subject: OU=openshift, CN=kube-csr-signer
        

        Issuer: OU=openshift, CN=kube-apiserver-to-kubelet-signer
        Validity
            Not Before: Mar  8 00:21:03 2023 GMT
            Not After : Mar  5 00:21:03 2033 GMT
        Subject: OU=openshift, CN=kube-apiserver-to-kubelet-signer
```